djangocms_admin_style/sass/components/_content.scss

```diff
@@ -3,6 +3,8 @@
 
 html {
     position: relative;
+    // make sure that scrollbar only appears if needed #270
+    overflow-y: auto;
```

Comment:
is it not the default behaviour of html?
(personally i prefer it to always be `scroll`)